Update convert-user-mailbox-to-shared-mailbox.md (#546)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md

    
6. For everything else you need to know about shared mailboxes, please see [Create a shared mailbox](create-a-shared-mailbox.md).

> ## Convert the mailbox using the office365 admin center

Reviewer: ```suggestion
## Convert the mailbox using the office365 admin center
```
```suggestion
## Convert the mailbox using the office 365 admin center
```

> - Mailboxes can be converted directly from the admin center as well

Reviewer: ```suggestion
- Mailboxes can be converted directly from the admin center as well.
```

::: moniker range="o365-worldwide"

> 1. Go to the [Microsoft 365 admin center](https://admin.microsoft.com/AdminPortal/Home#/homepage).

Reviewer 2: ```suggestion
1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.
```

::: moniker-end

::: moniker range="o365-germany"

> 1. Go to the [Office 365 admin center](https://portal.office.de/adminportal/home).

Reviewer 2: ```suggestion
1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=847686" target="_blank">Active users</a> page.
```

::: moniker-end

::: moniker range="o365-21vianet"

> 1. Go to the [Office 365 admin center](https://login.partner.microsoftonline.cn).

Reviewer 2: ```suggestion
1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=850628" target="_blank">Active users</a> page.
```

::: moniker-end

> 2. In the admin center, expand **Users** \> **Active Users**.

Reviewer 2: Delete this line

Author: This line is deleted now. We should be good to merge.

> 3. Choose the user whose mailbox nees to be converted

Reviewer: ```suggestion
3. Choose the user whose mailbox needs to be converted.
```

> 3. Choose the user whose mailbox needs to be converted.

Reviewer 2: ```suggestion
2. Choose the user whose mailbox you want to convert.
```

> 4. Expand Mail settings, Under **More Settings**, click **Convert to Shared mailbox**.

Reviewer: ```suggestion
4. Expand Mail settings. Under **More Settings**, click **Convert to Shared mailbox**.
```

> 4. Expand Mail settings, Under **More Settings**, click **Convert to Shared mailbox**.

Reviewer 2: ```suggestion
3. In the right pane, choose **Mail**. Under **More actions**, choose **Convert to shared mailbox**.
```

> 5. If the mailbox is smaller than 50GB, you can remove the [license from the user](../subscriptions-and-billing/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox. The shared mailbox needs it there as an anchor. If you are converting the mailbox of an employee that is leaving your organization, you should take additional steps to make sure that they cannot log in anymore. Please see [Remove a former employee from Office 365](../add-users/remove-former-employee.md).

Reviewer 2: ```suggestion
4. If the mailbox is smaller than 50GB, you can remove the [license from the user](../subscriptions-and-billing/remove-licenses-from-users.md), and stop paying for it. Don't delete the user's old mailbox. The shared mailbox needs it there as an anchor. If you are converting the mailbox of an employee that is leaving your organization, you should take additional steps to make sure that they cannot log in anymore. See [Remove a former employee from Office 365](../add-users/remove-former-employee.md).
```

> 6. For everything else you need to know about shared mailboxes, please see [Create a shared mailbox](create-a-shared-mailbox.md).

Reviewer 2: ```suggestion
5. For everything else you need to know about shared mailboxes, see [Create a shared mailbox](create-a-shared-mailbox.md).
```
